### PR TITLE
Update speech-to-text task with timestamps and streaming

### PR DESCRIPTION
**Pull request by jonatanklosko**
image

---

FTR we need changes to Nx and then a new Bumblebee to support longer audios with `Nx.Serving.run/2`, only then we will release a new version.

### CODE REVIEW

File: lib/kino_bumblebee/task_cell.ex [hunk 1]
```diff
@@ -1308,16 +1310,39 @@ defmodule KinoBumblebee.TaskCell do
...
+                    |> Integer.to_string()
+                    |> String.pad_leading(2, "0")
+
+                  seconds =
+                    seconds
+                    |> rem(60)
+                    |> Integer.to_string()
+                    |> String.pad_leading(2, "0")
+
+                  minutes <> ":" <> seconds
```
Comment (josevalim) [inline lib/kino_bumblebee/task_cell.ex:1338]:
```suggestion
                  seconds
                  |> round()
                  |> Time.from_seconds_after_midnight()
                  |> Time.to_string()
```

WDYT?

Comment (josevalim) [reply lib/kino_bumblebee/task_cell.ex:1338]: It does include hour as well, but it is less code, so I thought i would suggest

Comment (jonatanklosko) [reply lib/kino_bumblebee/task_cell.ex:1338]: Oh fantastic, we can do `|> String.replace_prefix("00:", "")`, wdyt?

Comment (josevalim) [reply lib/kino_bumblebee/task_cell.ex:1338]: Up to you, I would just keep it with hours and everything :) the less code, the better

**Pull request merged**